src/sedcli_main.c

```diff
@@ -504,6 +504,23 @@ static void print_blocksid_feat(struct sed_blocksid_supported_feat *blocksid)
 	sedcli_printf(LOG_INFO, "\tHardware Reset Flag : %d\n", blocksid->hardware_reset ? 1 : 0);
 }
 
+static void print_cnl_feat(struct sed_cnl_feat *cnl)
+{
+	sedcli_printf(LOG_INFO, "\nSED CNL FEATURES SUPPORTED\n");
+	sedcli_printf(LOG_INFO, "---------------------------\n");
+
+	sedcli_printf(LOG_INFO, "\tNamespace Non-Global Range Locking objects Supported : %s\n",
+			cnl->ranges_rsvd.range_c ? "Y" : "N");
+	sedcli_printf(LOG_INFO, "\tNamespace Non-Global Range Locking objects           : %s\n",
+			cnl->ranges_rsvd.range_p ? "ONE or MORE" : "ZERO");
+	sedcli_printf(LOG_INFO, "\tMaximum Key Count                                    : %d\n",
+			cnl->max_key_count);
+	sedcli_printf(LOG_INFO, "\tUnused Key Count                                     : %d\n",
+			cnl->unused_key_count);
+	sedcli_printf(LOG_INFO, "\tMaximum Ranges Per Namespace                         : %d (%#x)\n",
```

Comment:
I don't know %#x what this format prints.
I thought you want to print in hexadecimal form. If so use following format 0x%x

Reply:
%#x is same as 0x%x, the # flag appends 0x on it's own.

Reply:
HEre is output that I'm getting with your format specifier (I don't see "0x" added):

```
Maximum Ranges Per Namespace                         : 0 (0)

```

Reply:
Appends 0x for all non-zero values.